### DISCUSSION TITLE
Window 11

### DISCUSSION THREAD

**Discussion by ronsuds** [moneymanagerex/moneymanagerex#5483] [Q&A]
I am a relative newbie so please bear with me. Can MoneyManager EX run on Windows 11? Many thanks in advance!

**Comment by tactilis** [comment]
Yes. MMEX on Windows 11 no problems

**Comment by ronsuds** [comment]
Martin, I am having problems installing version 1.8.1. I would normally
post on the forum but I am not able to log in.

When I try to open version.8.1, I get multiple warnings about themes
(reverting to default theme, missing 92 icons, etc.). The final message
then states that the file may be corrupt and will not open.

I have used the color theme in the past. I believe this has something to do
with the calculator addition but I do not know how to add manually what may
be required.

Are you able to provide any direction?  Any assistance would be greatly
appreciated.

Ron Sutherland

**Comment by renato-mmex** [comment]
Did you answer yes or no to this question?

![image](https://github.com/user-attachments/assets/be272360-a1f6-4945-a8f8-cee62880ab21)

**Comment by ronsuds** [comment]
I think I initially said no and then tried yes the second time. No
difference. What should I be saying?

I think I will try the 32-byte version to see if that works.

Thanks for your help.

**Comment by renato-mmex** [comment]
I would delete “C:\Users\XXXXXXXX\AppData\Roaming\MoneyManagerEx\mmexini.db3”

XXXXXX = Username
